apps/sertra/pages/create.tsx

```diff
@@ -17,13 +17,45 @@ export const getStaticProps: GetStaticProps = async () => {
   };
 };
 
+type Inputs = {
+  serverName: string;
+  connectionInfo: string;
+  description: string;
+  mode: string;
+  mods: [string];
+  isPasswordProtected: string;
+};
+
 const SubmitServer: NextPage<{ swrFallback: { [key: string]: unknown } }> = ({
   swrFallback,
 }) => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Inputs>();
+  const onSubmit: SubmitHandler<Inputs> = (data) => {
+    const prepped_data = {
+      name: data.serverName,
+      description: data.description,
+      community: "v-rising",
+      connection_data: data.connectionInfo,
+      // TODO: Populate this with mod ids etc
+      mods: ["1", "2", "3"],
```

Comment:
When is tihs TODO going to be done and/or is it blocked by something?

Reply:
The populating is blocked by TS-712, by not having the actual mod data available. This is going to need an separate task, so I'm adding it to the inbox.

Format of the mods is currently:
`"mods":["ebkr-r2modman-3.1.31","Mythic-ServerLaunchFix-0.1.0","molenzwiebel-Wetstone-1.1.0","BepInEx-BepInExPack_V_Rising-1.0.0","ModLoader64-HuntControl-1.1.1","iZastic-AutoReady-1.0.1","iZastic-RemoveVignette-1.0.0","RPGMods-RPGMods-0.0.3"]`
I was just lazy and put 1 2 3 there 👍

Reply:
Not sure if the reply is for me or Vilppe or both. If you feel submitting numbers as strings is the way to go here, mark this comment resolved.